.gitignore

```diff
@@ -0,0 +1,28 @@
+**/**.ninja
```

Comment:
Is this not sufficient?

```
# build output directories
build
build*

# downloaded dependencies
third_party/googletest
third_party/turbojpeg
third_party/benchmark
tests/data
```

Reply:
Because it is uncertain whether the build directory is named 'build' 😂

Reply:
I have suggested these changes basing on this, https://github.com/google/jpegli/blob/main/.gitignore and https://github.com/google/heir/blob/main/.gitignore, thank you making them